tests/testthat/test_tracking.R

```diff
@@ -1,25 +1,17 @@
 context("Carbon tracking")
```

Comment:
@bpbond thoughts about adding carbon tracking to the old-new test?

Reply:
🤔 

I don't _think_ it's necessary? Here's my logic:
* As you note below, `oldnew` already covers carbon totals; it seems unlikely that the fractions could vary without the totals varying
* Tracking behavior (specifically, `fluxpool` behavior) is already covered by C++ unit test

Reply:
On a related note:

>do we want to change a carbon cycle parameter such as beta & ensure that when a parameter value changes the carbon tracking output also changes?

It's hard for me to imagine a case when this wouldn't be true, following the logic above. But I will add a test doing this.

Reply:
ah seems unnecessary sounds like it is good as written! 👍